src/ToggleButton/tests.jsx

```diff
@@ -0,0 +1,202 @@
+                expect( wrapper.find( Icon ).prop( 'size' ) ).to.equal( 'S' );
+            } );
+        } );
+    } );
+    describe( 'isDisabled', () =>
+    {
+        it( 'should be false by default', () =>
+        {
+            expect( instance.props.isDisabled ).to.be.false;
+        } );
+        it( 'should be passed to the button as "disabled" when true', () =>
+        {
+            wrapper.setProps( { isDisabled: true } );
+            expect( wrapper.prop( 'disabled' ) ).to.be.true;
+        } );
+    } );
+
+    describe( 'isReadOnly', () =>
+    {
+        it( 'should be false by default', () =>
+        {
+            expect( instance.props.isReadOnly ).to.be.false;
+        } );
+        it( 'should be passed to the button as "readOnly" when true', () =>
+        {
+            wrapper.setProps( { isReadOnly: true } );
+            expect( wrapper.prop( 'readOnly' ) ).to.be.true;
+        } );
+    } );
+
+
+    describe( 'onClick', () =>
+    {
+        it( 'should be undefined by default', () =>
+        {
+            expect( instance.props.onClick ).to.be.undefined;
+        } );
+
+        it( 'should be passed to the button element', () =>
+        {
+            const onClick = () =>
+            {};
+            wrapper.setProps( { onClick } );
+            expect( wrapper.prop( 'onClick' ) ).to.be.equal( onClick );
+        } );
+    } );
+
+    describe( 'onFocus', () =>
+    {
+        it( 'should be undefined by default', () =>
+        {
+            expect( instance.props.onFocus ).to.be.undefined;
+        } );
+        it( 'should be passed to the button element', () =>
+        {
+            const onFocus = () =>
+            {};
+            wrapper.setProps( { onFocus } );
+            expect( wrapper.prop( 'onFocus' ) ).to.be.equal( onFocus );
+        } );
+    } );
+
+    describe( 'onMouseOver', () =>
+    {
+        it( 'should be undefined by default', () =>
+        {
+            expect( instance.props.onMouseOver ).to.be.undefined;
+        } );
+        it( 'should be passed to the button element', () =>
+        {
+            const onMouseOver = () =>
+            {};
+            wrapper.setProps( { onMouseOver } );
+            expect( wrapper.prop( 'onMouseOver' ) ).to.be.equal( onMouseOver );
+        } );
+    } );
+
+    describe( 'onMouseOut', () =>
+    {
+        it( 'should be undefined by default', () =>
+        {
+            expect( instance.props.onMouseOut ).to.be.undefined;
+        } );
+        it( 'should be passed to the button element', () =>
+        {
+            const onMouseOut = () =>
+            {};
+            wrapper.setProps( { onMouseOut } );
+            expect( wrapper.prop( 'onMouseOut' ) ).to.be.equal( onMouseOut );
+        } );
+    } );
+} );
+
+describe( 'ToggleButtonDriver', () =>
+{
+    let wrapper;
+    beforeEach( () =>
+    {
+        wrapper = mount( <ToggleButton /> );
+    } );
+    describe( 'onClick', () =>
```

Comment:
This `describe` block is testing the `toggle` driver function, so this line should be:
```
describe( 'toggle', () =>
```